core/math/expression.h

```diff
@@ -161,17 +146,15 @@ class Expression : public Reference {
 	};
 
 	struct ConstantNode : public ENode {
-		Variant value = Variant::NIL;
+		Variant value;
```

Comment:
IINM `Variant` has a default constructor so we don't need to initialize it this way.